### PR TITLE
Tickets/preops 5180

### PR DESCRIPTION
**Pull request by womullan**

### CODE REVIEW

File: system.tex [hunk 1]
```diff
@@ -17,31 +19,39 @@ \subsection{Data verification and quality assurance }
...
-These team members will have to have \gls{SLAC} accounts to perform this work - \gls{SLAC} accounts are governed by (need doc).\action{RD}{Mar 2022}{Handle for doc covering users accounts at \gls{SLAC}}.
+These team members will have to have \gls{SLAC} accounts to perform this work - \gls{SLAC} accounts are governed by SLAC procedures \cite{SLACOB} and \cite{SLACNH}.
 In addition they will need summit accounts as governed by \citeds{ittn-045} and \citeds{ittn-010}.
 
 
 
 \subsection{Data processing}
 \subsubsection{Data Release Processing}
-DRP is performed only on unencumbered data which is fully available to Rubin Data Access rights holders.
+DRP is performed only on unencumbered data which is fully available to Rubin data rights holders.
```
Comment (rblum5) [inline system.tex]:
```suggestion
DRP is performed only on unembargoed data which is fully available to Rubin data rights holders.
```

File: system.tex [hunk 2]
```diff
@@ -17,31 +19,39 @@ \subsection{Data verification and quality assurance }
...
+\begin{itemize}
+\item SLAC lists cyber policies on \url{https://it.slac.stanford.edu/cybersecurity/compliance}.
+\item UKDF falls under the \gls{IRIS} security policies listed on \url{https://www.iris.ac.uk/security/}.
+\item FrDF lists cyber policies and compliance on \url{https://doc.lsst.eu/cybersecurity/cybersecurity.html}.
+\end{itemize}
+
 
 
 \subsection{Data access}
 After a short period of encumbrance image and derived data are made available to all
```
Comment (rblum5) [inline system.tex]: Please clarify what encumbrance means.

File: system.tex [hunk 3]
```diff
@@ -17,31 +19,39 @@ \subsection{Data verification and quality assurance }
 They require access to tools installed at the summit as well as in the data facilities.
 In particular they will need access to images as they are processed to generate alerts.
 
-These team members will have to have \gls{SLAC} accounts to perform this work - \gls{SLAC} accounts are governed by (need doc).\action{RD}{Mar 2022}{Handle for doc covering users accounts at \gls{SLAC}}.
+These team members will have to have \gls{SLAC} accounts to perform this work - \gls{SLAC} accounts are governed by SLAC procedures \cite{SLACOB} and \cite{SLACNH}.
 In addition they will need summit accounts as governed by \citeds{ittn-045} and \citeds{ittn-010}.
 
 
 
 \subsection{Data processing}
```
Comment (rblum5) [inline system.tex:27]: I am confused why we skip prompt processing and go straight to DRP. PP is the more relevant for security.

File: body.tex [hunk 4]
```diff
@@ -23,13 +23,13 @@ \section{Introduction}
 After an agreed embargo period all data will be transmitted to the \gls{IN2P3} in Lyon, France, and at least some data is transmitted to the Royal Observatory Edinburgh.
 Processing occurs in all three locations using a coordinate release of the Rubin \gls{Science Pipelines}.
```
Comment (rblum5) [inline body.tex]:
```suggestion
Processing occurs in all three locations using a coordinated release of the Rubin \gls{Science Pipelines}.
```

**Review by rblum5 — COMMENTED** [review]
Blum review complete

File: system.tex (hunk 2 above)
Comment (womullan) [reply system.tex]: should use embargo .. will change

File: system.tex (hunk 3 above)
Comment (womullan) [reply system.tex:27]: added a section for that .. good catch

**Pull request merged**